WalletWasabi.Fluent/Models/Wallets/WalletModel.cs

```diff
@@ -19,6 +19,7 @@ namespace WalletWasabi.Fluent.Models.Wallets;
 public partial class WalletModel : ReactiveObject, IWalletModel
 {
 	private readonly TransactionHistoryBuilder _historyBuilder;
+	private readonly Lazy<IWalletBalancesModel> _balances;
 	private readonly Lazy<IWalletCoinjoinModel> _coinjoin;
 
 	public WalletModel(Wallet wallet)
```

Comment:
Altough on initalization, one WalletModel is created for each wallet now,
when clicking on the Wallet Info button in an open wallet
![image](https://github.com/zkSNACKs/WalletWasabi/assets/54551879/fa98b73b-115c-4091-9a0d-89b36c1011b8)
This code will run
https://github.com/zkSNACKs/WalletWasabi/blob/a0752a7be33e2e10d31ca50127f0921cdeb5756d/WalletWasabi.Fluent/ViewModels/Wallets/WalletViewModel.cs#L110-L125

This creates a new WalletModel every time we open the Wallet Info.
Not sure if it's a big problem or not.

Reply:
@Szpoti That code uses `WalletRepository.CreateWalletModel()` which is safe, so it's safe as well.